articles/iot-central/howto-set-up-template.md

```diff
@@ -123,6 +124,26 @@ If the device sends too many data points in a small duration, the state measurem
 > [!NOTE]
 > The data type of the state measurement is string.
 
+### Create a Location Measurement
+
+To add a new location measurement, select **+ New**, choose **Location** as the measurement type, and enter the details on the form.
+
+For example, you can add a new location telemetry measurement:
+
+| Display Name        | Field Name    |
+| --------------------| ------------- |
+| NW Territory        | nwterritory   |
+
+!["Create Location" form with details for location measurement](./media/howto-set-up-template/measurementsform.png)
+
+After you select **Save**, the **Location** measurement appears in the list of measurements. In a short while, you see the visualization of the location data from the simulated device.
+
+When displaying location, you can choose from the following options: latest location and location history. **Location history** is only applied over the selected time range.
+
+> [!NOTE]
+> The data type of the location measurement is longitude, latitude and the option of altitude.
```

Comment:
Can we provide the actual payload schema? I think we need to do this for all the other measurements as well but just specifying Lon, Lat and Alt will not be enough. It is an object so we should call that out.

Reply:
Have been resolved with latest commit. Added an example of the structure.